README.md

```diff
@@ -53,19 +53,22 @@ Clone the docs repo into a directory you want the ODK-X docs files to be located
 1. Visit this link https://github.com/odk-x/docs on your browser to locate the docs repo
 2. Find and click the "fork" button to create a personal fork of the project on Github
 3. Navigate to the directory you want the files to be located using the "cd" (Change Directory) command on a command-line interface on your local machine:
+
 ```
 cd <DIRECTORY>
 ```
-4. Get a copy of the ODK-X docs repository on your local machine by cloning the forked repo to your local machine from your github account using the clone command below. Replace "LINK-TO-YOUR-FORKED-REPO" below with the actual link to your forked repo:  
+
+4. Copy the forked ODK-X docs repository URL by clicking on the green "Code" button on Github and navigate to the command-line interface
```

Comment:
There are my graphical interface programs that you can use and could be easier. You should add in parenthesis that "if you do not want to use the command line you can use use Git graphical source control programs."

Reply:
> There are my graphical interface programs that you can use and could be easier. You should add in parenthesis that "if you do not want to use the command line you can use use Git graphical source control programs."

Hi, @wbrunette Thanks for pointing that out. I realized the instruction in the README file is to be used on the command line and not on the Git graphical source control, which I added to the file, thus making it conflicting. 

However, I do not know exactly where the clause **"if you do not want to use the command line you can use use Git graphical source control programs."** will fit in.  

Please, can you indicate where you would want me to add that.

Reply:
Text:
4. Copy the forked ODK-X docs repository URL by clicking on the green "Code" button on Github and navigate to the command-line interface. (As an alternative to using the command-line interface, there are many graphical source control programs that allow you to interact with Git repositories more easily).

ALSO:
You may want to add local machine so people do not try to find the command-line on GitHub

Reply:
@wbrunette Please could you review. I effected the changes.